migrate-mongo-config.js

```diff
@@ -0,0 +1,34 @@
+// In this file you can configure migrate-mongo
```

Comment:
Use a typescript file, always. We can run this with `ts-node` in the npm script section